tools/tck-integrity-tests/src/test/scala/org/opencypher/tools/tck/ValidateSteps.scala

```diff
@@ -135,6 +137,20 @@ trait ValidateSteps extends AppendedClues with Matchers with OptionValues with V
       case _ => succeed
     }
 
+    withClue("scenario does not declare conflicting parameters") {
+      val parameters = steps.collect {
```

Comment:
I guess this would need to be `flatMap` or the like, so that we do not end up with a `List[List[Parameter]]`.